src/setting.h

```diff
@@ -11,8 +11,21 @@
 #include <utility>
 #include <vector>
 
+// This is here for compatibility with older versions of Visual Studio
```

Comment:
Could you extract this into an new .h file so you don't have to keep declaring it? It was probably fine when there were two instances of this, but not so many.

Reply:
Sure! Will `include/yaml-cpp/noexcept_def.h` be fine?